templates/default/event/eventmodify.html

```diff
@@ -114,7 +114,7 @@ <H1>{t a=$event.ticketid}Add new event to request #$a{/t}</H1>
 					{/box_row}
 
 					{box_row icon="user" label="Users:" field_class="multiselect" icon_class="fa-fw"}
-						<SELECT name="event[userlist][]" class="lms-ui-multiselect" id="userlist" multiple data-default-value="{trans("- select user -")}" data-shorten-to-default-value="false">
+						<SELECT name="event[userlist][]" class="lms-ui-multiselect" id="userlist" multiple data-default-value="{trans("- select user -")}" data-shorten-to-default-value="false" {if !empty(ConfigHelper::checkConfig('phpui.event_user_required'))}required{/if}>
```

Comment:
Dlaczego tego selecta nie rozbijesz na kilka wierszy?

Reply:
Już rozbiłem. Mi się mięści to nawet w vim'ie na konsoli dlatego o tym zapominam regularnie. Może dodasz do travisa regułę?